pages/common/less.md

```diff
@@ -1,6 +1,7 @@
 # less
 
 > Open a file for interactive reading, allowing scrolling and search.
+> More information: <https://greenwoodsoftware.com/less/>.
```

Comment:
Should we remove slashes at the end or leave them? IMO, it looks better without.

Reply:
I prefer to avoid the redirection just by 1 slash, but there are redirects in the tldr links

Reply:
A discussion is in order on when and how much to simplify links; it's been bugging me a lot.